### PR TITLE
DOC readme expansion

### PR DESCRIPTION
**Pull request by adrinjalali**
A bit of more README

fixes https://github.com/skops-dev/skops/issues/77

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -26,11 +26,19 @@
...
 =====
 
-This library helps you share your scikit-learn based models and put them in
-production.
+``skops`` is a Python library helping you share your `scikit-learn
+<https://scikit-learn.org/stable/>`__ based models and put them in production.
+At the moment, it includes the following features:
+
+- ``skops.hub_utils``: tools to create a model repository to be stored on
+  `Hugging Face Hub <https://hf.co/models>`__, mainly through
```
Comment (BenjaminBossan) [inline README.rst:34]: > mainly through ``skops.hub_utils.init`` and ``skops.hub_utils.push``.

I think this doesn't add much for the reader. I think the added value (apart from storing and sharing) rather comes from the inference widget. WDYT?

File: README.rst [hunk 2]
```diff
@@ -26,11 +26,19 @@
...
+  ``skops.hub_utils.init`` and ``skops.hub_utils.push``.
+- ``skops.card``: tools to create a model card explaining what the model does
```
Comment (BenjaminBossan) [inline README.rst:36]: How about adding something along the lines of "automatically populating" the model card? I know not everything is populated automatically, but some stuff is, which saves the user time.

**Review by BenjaminBossan — APPROVED** [review]
Good addition. I have two suggestions, up to you if you want to follow them.

**Comment by adrinjalali** [comment]
Does this look better now?

**Comment by BenjaminBossan** [comment]
Yes, I think it's much easier now for users to understand the benefit. Ideally, we can later get someone who doesn't know anything about skops to read it and give their feedback, since all of us are already biased.

I can merge now if or wait for @merveenoyan to review.

**Comment by adrinjalali** [comment]
I think you could merge, not a big change here I think.

**Pull request merged**